src/query/plan/operator.hpp

```diff
@@ -1120,6 +1124,45 @@ class Filter : public memgraph::query::plan::LogicalOperator {
   std::shared_ptr<memgraph::query::plan::LogicalOperator> input_;
   std::vector<std::shared_ptr<memgraph::query::plan::LogicalOperator>> pattern_filters_;
   Expression *expression_;
+  const memgraph::query::plan::Filters all_filters_;
+
+  static std::string SingleFilterName(const query::plan::FilterInfo &single_filter) {
```

Comment:
This is good but I think even better would be that this is a method of FilterInfo struct to avoid static methods, seems like a bit better design

Reply:
That makes sense! I've considered it, but decided against doing so for the following reasons:
* `SingleFilterName` is used only by `Filter::ToString`
* `ExpandVariable` has an analogous (but non-static) `OperatorName` method